views/pages/organisation/team.html

```diff
@@ -20,6 +21,24 @@
 
       <div class="govuk-grid-column-two-thirds">
 
+        {% set status = '' %}
+        {% set html = '' %}
+
+        {% if addedUser %}
+          {% if userFullName %}
+            {% set status = "success" %}
+            {% set html = "<h2>You have successfully added " + userFullName %}
+          {% else %}
+            {% set status = "failure" %}
+            {% set html="<h2>There was a problem adding user</h2>" %}
```

Comment:
Can we tighten the grammar here?

```suggestion
            {% set html="<h2>There was a problem adding the user</h2>" %}
```

Alternatively, if we reach a dead-end with the "adding a user that already exists" problem I commented on above, we could expand this to "There was a problem adding the user. The user may already exist in LEV". This is something we should validate with Jo first though.

Reply:
The error message is changed as discussed with Jo